packages/create-sitecore-jss/src/templates/react/src/index.js

```diff
@@ -9,6 +9,14 @@ import AppRoot from './AppRoot';
 import GraphQLClientFactory from './lib/GraphQLClientFactory';
 import config from './temp/config';
 import i18ninit from './i18n';
+import { stopReportingRuntimeErrors } from 'react-error-overlay';
+
+/* as we handle and show errors via error boundaries and console we want to avoid error overlay showing up */
+/* in part because it's not working correctly at the moment: https://github.com/facebook/create-react-app/issues/11773 */
+/* please feel free to disable this if overlay is needed for your dev environments */
+if (process.env.NODE_ENV === 'development') {
+  stopReportingRuntimeErrors();
+}
```

Comment:
What's the implication if this isn't done? If it is just extra error notification, then might be willing to accept that to lose the added dependency to manage on `react-error-overlay`.

Reply:
react-error-overlay is already added via react itself - in here it's just a direct import
without this users would see an iframe with error description in connected mode, taking up the whole page for react apps.